hedgehog/src/Hedgehog/Internal/TH.hs

```diff
@@ -21,14 +21,38 @@ import           Language.Haskell.TH.Syntax
 type TExpQ a =
   Q (TExp a)
 
--- | Check all the properties in a file.
+-- | Check all the properties in a file sequentially.
+--
+-- > tests :: IO Bool
+-- > tests =
+-- >   $$(checkSequential)
+--
+checkSequential :: TExpQ (IO Bool)
```

Comment:
I guess you'd be able to better distribute the load that way, but we'd need a way to mark a test a concurrent or sequential, maybe per property is a better approach anyway